raiden-dapp/tests/e2e/utils/blockchain-interaction.ts

```diff
@@ -33,11 +33,9 @@ export function mintConnectedTokenFromSelectHubScreen() {
 export function openChannel() {
   cy.log('open channel');
   // cypress selectors: raiden-dapp/src/views/OpenChannelRoute.vue
```

Comment:
Maybe delete this comment as well.

Reply:
We have tons of these comments all over the place. I think deleting just this one as well won't be helpful. We might wanna open an issue to finally remove all of them. 😬